README.md

```diff
@@ -42,13 +43,15 @@ var MyComponent = onClickOutside(React.createClass({
     // ...handling code goes here...
   },
   ...
-}), function (instance) {
-    instance.myClickOutsideHandler
+}), {
+  handleClickOutside: function(instance) {
```

Comment:
I could not find a way to use the `handleClickOutside: Component.myClickOutsideHandler`, so I stuck with the accessor function notation.